examples/ee-demo/app.js

```diff
@@ -68,7 +81,12 @@ export default class App extends React.Component {
     const layers = this.state.eeImage && [
       new EarthEngineLayer({
         eeObject: this.state.eeImage,
-        visParams: {min: 0, max: 255},
+        // visParams: {min: 0, max: 255},
```

Comment:
ee.state.visParams?

Reply:
`visParams` doesn't exist in `state` in the example app.

Reply:
Well, I was suggesting adding `state.visParams`. Just so you could do all the configuration in one place and not have if statements all over the app (when implementing support for all three object types).